<?xml version="1.0"?>

<launch>

  <arg name="robot_name" default="fetch_"/>
  <arg name="robot_num" default="3"/>

  <node pkg="robot_position_visualizer" name="RobotPositionVisualizer" type="RobotPositionVisualizer.py" output="screen">
    <rosparam param="/robot_name" subst_value="True">$(arg robot_name)</rosparam>
    <rosparam param="/robot_num" subst_value="True">$(arg robot_num)</rosparam>
  </node>

</launch>

